Source/ASNetworkImageNode.mm

```diff
@@ -606,13 +593,13 @@ - (void)_downloadImageWithCompletion:(void (^)(id <ASImageContainerProtocol> ima
 
 - (void)_lazilyLoadImageIfNecessary
 {
-  __instanceLock__.lock();
+  [self lock];
     __weak id<ASNetworkImageNodeDelegate> delegate = _delegate;
     BOOL delegateDidStartFetchingData = _delegateFlags.delegateDidStartFetchingData;
     BOOL isImageLoaded = _imageLoaded;
     NSURL *URL = _URL;
     id currentDownloadIdentifier = _downloadIdentifier;
-  __instanceLock__.unlock();
+  [self unlock];
```

Comment:
The difference in style between these statements and the macros feels a bit less prominent / conspicuous than before, where all locks involved __instanceLock__.

I wonder if we should use a macro for even the simple lock / unlock message send, just to make locking operations consistent? Even if we did that, lock / unlock would still be available, especially for third party code.

Reply:
It's true – I'm iffy on this. I totally see your point that there's a mismatch in look and feel, but I don't like when there's two ways to do the same thing, and I like the referential transparency.

I also kind of like the mismatch, since it "discourages" use of these methods by making them stick out.

Let's sleep on this. There will be more locking diffs in the future and we can add the macros easily at any time.